manifests/resource-topology-exporter-ds.yaml

```diff
@@ -54,32 +54,25 @@ spec:
       - name: resource-topology-exporter-container
         image: quay.io/swsehgal/resource-topology-exporter:latest
         command:
-        - /usr/local/bin/resource-topology-exporter
-        - --watch-namespace=rte
+        - /bin/resource-topology-exporter
+        - --export-namespace=default
```

Comment:
I see in the  manifest directory we have test pods requesting SR-IOV devices in the `rte` namespace, so think it would be better to provide `--watch-namespace=rte` here as well.

Reply:
sure, added